src/contracts/SafetyNet.sol

```diff
@@ -441,6 +460,7 @@ contract SafetyNet is ISafetyNet, ReentrancyGuard, OwnableUpgradeable {
         revert ExceedsSmallWithdrawalLimit();
       }
       memberWithdrawableBalance[_id][_member] -= _withdrawAmount;
+      safetyNetBalance[_id] -= _withdrawAmount;
```

Comment:
damn we really didn't have this before and none of the tests caught it?

Reply:
definitely create an issue for testing this if we are not. this is a pretty bad one to have slipped through